shopinvader_delivery_pickup_colissimo_widget/controllers/main.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 ACSONE Sa/ (http://www.acsone.eu)
+# Cédric Pigeon <cedric.pigeon@acsone.eu>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from odoo.addons.shopinvader.controllers import main
+from odoo.http import route
+
+_logger = logging.getLogger(__name__)
+
+
+class InvaderController(main.InvaderController):
+    @route(["/shopinvader/cart/get_laposte_dropoff_site_token"],
```

Comment:
@sebastienbeau This route seems to call an undefined method. Moreover this method should go into a pickup service :smirk: